### ISSUE TITLE
Message box saying "Got error while processing diagnostics: Failing due to -Werror." in Sublime

### ISSUE THREAD

**Issue by Pastafarianist** [haskell/haskell-ide-engine#662]
I am using Sublime and I am working with a codebase I cannot edit freely for _reasons_. Anyway, while browsing code, often a message box pops up that says

```
Got error while processing diagnostics: 
Failing due to -Werror.
```

This is annoying and useless. It would be much more preferable to see a specific list of warnings in the box at the bottom of the window.

I'm not sure though whether the issue is in HIE or in Sublime LSP client, so if this is the wrong issue tracker to post this, I will move it to the LSP client tracker.

**Comment by Pastafarianist** [comment]
Looks like a dupe of https://github.com/haskell/haskell-ide-engine/issues/391.

**Comment by alanz** [comment]
This should be fixed via #817
